### PR TITLE
feat: create user model and json

### PR DESCRIPTION
**Pull request by DobrovskayaElena**

### CODE REVIEW

File: src/app/core/models/user.model.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface IUser {
+    userId?: string;
+    email?: string;
+    password?: string;
+    name?: string;
+    surname?: string;
+    role?: string;
```
Comment (nattallius) [inline src/app/core/models/user.model.ts]: What it the reason to make all fields optional?

**Review by nattallius — APPROVED** [review]

File: src/app/core/models/user.model.ts (hunk 1 above)
Comment (DobrovskayaElena) [reply src/app/core/models/user.model.ts]: Probably, I made a mistake :)

**Review by Zolotou — APPROVED** [review]
Why you choose to rid of the "?".

**Comment by DobrovskayaElena** [comment]
> Why you choose to rid of the "?".

I decided that all fields are important :)

**Pull request merged**